### PR TITLE
Add default routes k8s windows

### PR DESCRIPTION
**Pull request by paulyufan2**
**Reason for Change**:

Add default routes k8s windows

In Swiftv2 Windows scenario, we need to set default route on delegatedNIC interface instead of infraNIC interface, so we need to set skpDefaultRoutes to true for infraNIC interface

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: cns/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,9 @@ FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635
 # intermediate for win-ltsc2022
 FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as ltsc2022
 
+# FROM mcr.microsoft.com/windows/server/insider:10.0.26080.1 as
```
Comment (kmurudi) [inline cns/Dockerfile:31]: remove L31

File: cns/middlewares/k8sSwiftV2_windows_test.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package middlewares
+
+import (
+	"testing"
```
Comment (jpayne3506) [inline cns/middlewares/k8sSwiftV2_windows_test.go:4]: Is this a package..?

Comment (paulyufan2) [reply cns/middlewares/k8sSwiftV2_windows_test.go:4]: yes, it's golang test package

File: cns/middlewares/k8sSwiftV2_windows.go [hunk 3]
```diff
@@ -6,7 +6,12 @@ import (
 )
 
 // setRoutes sets the routes for podIPInfo used in SWIFT V2 scenario. This is a no-op as route setting is not applicable for Windows.
```
Comment (jpayne3506) [inline cns/middlewares/k8sSwiftV2_windows.go:8]:
```suggestion
// setRoutes sets the routes for podIPInfo used in SWIFT V2 scenario. 
```

File: cns/middlewares/k8sSwiftV2_windows_test.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+	"github.com/Azure/azure-container-networking/cns"
+	"github.com/Azure/azure-container-networking/cns/middlewares/mock"
+	"gotest.tools/v3/assert"
+)
+
+func TestSetRoutesSuccess(t *testing.T) {
```
Comment (jpayne3506) [inline cns/middlewares/k8sSwiftV2_windows_test.go:11]: Do we want this test to be ran in our PR pipeline? Currently the only windows unit tests being run are npm.

CC: @huntergregory @matmerr

**Review by jpayne3506 — CHANGES_REQUESTED** [review]

File: cns/middlewares/k8sSwiftV2_windows_test.go (hunk 4 above)
Comment (jpayne3506) [reply cns/middlewares/k8sSwiftV2_windows_test.go:11]: #1554

Comment (huntergregory) [reply cns/middlewares/k8sSwiftV2_windows_test.go:11]: It's ok with us (NPM team) if you run these CNS UTs in the pipeline job for Windows UTs (which only runs NPM tests right now).

**Review by jpayne3506 — DISMISSED** [review]
lgtm, would love to see the inclusion of CNS UT(s) on the PR pipeline.

File: cns/middlewares/k8sSwiftV2_windows.go [hunk 5]
```diff
@@ -5,8 +5,12 @@ import (
 	"github.com/Azure/azure-container-networking/cns/logger"
 )
 
-// setRoutes sets the routes for podIPInfo used in SWIFT V2 scenario. This is a no-op as route setting is not applicable for Windows.
-func (k *K8sSWIFTv2Middleware) setRoutes(_ *cns.PodIpInfo) error {
-	logger.Printf("[SWIFTv2Middleware] setRoutes is a no-op on Windows")
+// for AKS L1VH, do not set default route on infraNIC to avoid customer pod reaching all infra vnet services
+// default route is set for secondary interface NIC(i.e,delegatedNIC)
+func (k *K8sSWIFTv2Middleware) setRoutes(podIPInfo *cns.PodIpInfo) error {
+	logger.Printf("[SWIFTv2Middleware] setRoutes: only set skipDefaultRoutes flag to true for InfraNIC")
```
Comment (rbtr) [inline cns/middlewares/k8sSwiftV2_windows.go:11]: this is not a useful log, as it is simply describing the code (the previous log was also not a useful log).
the log should tell us _what happened_. so at a minimum it needs to be moved inside and/or after the conditional, to give us that information about what decision was made and why.

**Review by rbtr — CHANGES_REQUESTED** [review]

**Review by rbtr — APPROVED** [review]

**Pull request merged**